src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -161,6 +165,44 @@ describe('manage nsfs cli bucket flow', () => {
             path_exists = await is_path_exists(DEFAULT_FS_CONFIG, bucket_temp_dir_path);
             expect(path_exists).toBe(false);
         });
+
+        it('cli delete bucket not fail with force flag, when bucket path is not empty', async () => {
+            let action = ACTIONS.ADD;
+            const bucket_options = { name: "bucket4", owner: bucket_defaults.owner, path: bucket_defaults.path };
+            // write the json_file_options
+            const path_to_option_json_file_name = await create_json_bucket_options(path_to_json_bucket_options_dir, bucket_options);
+            const command_flags = {config_root, from_file: path_to_option_json_file_name};
+            // create the bucket and check the details
+            await exec_manage_cli(TYPES.BUCKET, action, command_flags);
+            const bucket = await read_config_file(config_root, CONFIG_SUBDIRS.BUCKETS, "bucket4");
```

Comment:
Why are you using the from-file bucket creation?
I think it is better to create the bucket using the command either in the `beforeEach`.